packages/flutter/lib/src/material/dialog.dart

```diff
@@ -219,6 +219,84 @@ class Dialog extends StatelessWidget {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=freeform}
```

Comment:
Instead of the `freeform` template, can you use another one to reduce the amount of boiler plate? `stateful_widget_scaffold_center` might be a good one. :)

Reply:
Ok